src/Nethermind/Nethermind.Hive.ConsensusWorkflowGenerator/Program.cs

```diff
@@ -0,0 +1,224 @@
+using System.Text;
+
+namespace Nethermind.Hive.ConsensusWorkflowGenerator;
+
+public static class Program
+{
+    const long MaxSizeWithoutSplitting = 35_000_000;
+    const long TargetSize = 23_000_000;
+    const long PenaltyForAdditionalInit = 300_000;
+
+    static void Main(string[] args)
+    {
+        StringBuilder fileContent = new();
+
+        List<string> directories = GetTestsDirectories();
```

Comment:
Would make sense to take some parameter from command line about the path to tests